nilearn/decoding/space_net.py

```diff
@@ -720,7 +720,7 @@ def fit(self, X, y):
         X = self.masker_.fit_transform(X)
 
         X, y = check_X_y(X, y, ['csr', 'csc', 'coo'], dtype=np.float,
-                         multi_output=True, y_numeric=True)
+                         multi_output=True, y_numeric=not self.is_classif)
```

Comment:
Seems unrelated, but correct.

Reply:
It's actually not unrelated: the changes that we did implied that we were passing in non numerical values for y, and hence the code started failing where it shoudn't :).